homeassistant/components/venstar/climate.py

```diff
@@ -192,6 +193,7 @@ def extra_state_attributes(self):
         return {
             ATTR_FAN_STATE: self._client.fanstate,
             ATTR_HVAC_STATE: self._client.state,
+            ATTR_SCHED_PART: self._client.get_info("schedulepart"),
```

Comment:
What does the schedule part represent? Is it a time integer?

Reply:
I checked the docs PR and it represents one of the available schedule times, which are parts of a say.

I think this should be an `enum` device class sensor instead.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
I think you're correct on this choice. I have pushed changes to create an enum entity for the schedule part.

I will update the corresponding documentation PR once this is confirmed to be the correct path to take.